spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :feature do
+  before do
+    @user1 = User.create!(id: 5, name: 'Khaing',
+                          photo: 'https://c0.klipartz.com/pngpicture/695/991/gratis-png-profesor.png', bio: 'officer')
+    @user2 = User.create!(id: 6, name: 'May',
+                          photo: 'https://c0.klipartz.com/pngpicture/695/991/gratis-png-profesor.png', bio: 'student')
+    Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    Post.create!(title: 'Intro1', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    @post1 = Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user2)
+    @comment1 = Comment.create(post: @post1, user: @user2, text: 'My comment')
+    @like1 = Like.create(post: @post1, user: @user2)
+  end
+
+  describe 'returns post show page' do
+    it "returns author's name" do
+      visit user_posts_path(@user2)
+      expect(@user2.name).to have_content('May')
+    end
+
+    it 'returns username of each commentor.' do
+      comment = @user2.posts[0].comments.first
+      expected_username = 'May'
+
+      expect(comment.user.name).to eq(expected_username)
+    end
```

Comment:
- [ ] In this test you should map through all comments and asserts that you can see the username of each comment, not only check for the username of one comment. Please fix that.